GetandTake/DataAccess/NorthwindDbContext.cs

```diff
@@ -1,14 +1,23 @@
 ﻿using GetandTake.Models;
 using Microsoft.EntityFrameworkCore;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.AspNetCore.Identity;
 
 namespace GetandTake.DataAccess;
 
-public class NorthwindDbContext : DbContext
+public class NorthwindDbContext : IdentityDbContext<IdentityUser>
 {
-    public NorthwindDbContext(DbContextOptions<NorthwindDbContext> options) : base(options)
+    public NorthwindDbContext(
+        DbContextOptions<NorthwindDbContext> options) : base(options)
     {
+    }
 
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+        modelBuilder.ApplyConfigurationsFromAssembly(typeof(NorthwindDbContext).Assembly);
+        base.OnModelCreating(modelBuilder);
```

Comment:
I strongly advise to put call of base method at the beginning since it may override your custom configs (based on my own experience, had to spent about an hour to determine why it was not working)